test.py

```diff
@@ -523,7 +523,7 @@ def build_test_list(self) -> List[str]:
         """For pytest, search for directories recursively"""
         path = self.suite_path
         pytests = itertools.chain(path.rglob("*_test.py"), path.rglob("test_*.py"))
-        tests = [os.path.splitext(t.relative_to(self.suite_path))[0] for t in pytests]
+        tests = [str(t.relative_to(self.suite_path)) for t in pytests]
```

Comment:
I am not against adding ".py" to test file names in the command (this is what pytest does so I'm used to it), but please make sure you update the examples in the documentation (doc/dev/testing.md) and commit messages to include the ".py" that is now needed (if I understand correctly).